config/xml_schemas/config_batch.xsd

```diff
@@ -57,9 +58,12 @@
         <xs:element minOccurs="0" ref="jobid_pattern"/>
 
   <!-- depend_string: How to indicate to batch system that this job should not run until
-       a previous job has completed successfully -->
+       a previous job has completed (possibly unsuccessfully) -->
```

Comment:
Was there any discussion and agreement regarding changing this functionality?  I don't recall it.